modules/govuk/manifests/deploy/config.pp

```diff
@@ -92,31 +92,27 @@
 
     'ERRBIT_ENVIRONMENT_NAME': value   => $errbit_environment_name;
     'SENTRY_CURRENT_ENV': value        => $errbit_environment_name;
-    'GOVUK_APP_DOMAIN': value          => $app_domain;
     'GOVUK_ASSET_HOST': value          => $asset_root;
     'GOVUK_ASSET_ROOT': value          => $asset_root;
     'GOVUK_WEBSITE_ROOT': value        => $website_root;
   }
 
-  # TODO: Set some internal services specifically before we've figured out
-  # how to entirely use internal domains
   if $::aws_migration {
     $app_domain_internal = hiera('app_domain_internal')
 
     govuk_envvar {
-      'GOVUK_APP_DOMAIN_INTERNAL': value        => $app_domain_internal;
-
-      # These variables are manually set in the draft stack (e.g. s_draft_cache),
-      # make sure they're separated out in those locations otherwise puppet
-      # won't run cleanly.
-      'PLEK_SERVICE_CONTENT_STORE_URI': value   => "https://content-store.${app_domain_internal}";
-      'PLEK_SERVICE_EMAIL_ALERT_API_URI': value => "https://email-alert-api.${app_domain_internal}";
-      'PLEK_SERVICE_LICENSIFY_URI': value       => "https://licensify.${licensify_app_domain}";
-      'PLEK_SERVICE_MAPIT_URI': value           => "https://mapit.${app_domain_internal}";
-      'PLEK_SERVICE_PUBLISHING_API_URI': value  => "https://publishing-api.${app_domain_internal}";
-      'PLEK_SERVICE_RUMMAGER_URI': value        => "https://rummager.${app_domain_internal}";
-      'PLEK_SERVICE_SEARCH_URI': value          => "https://search.${app_domain_internal}";
-      'PLEK_SERVICE_STATIC_URI': value          => "https://static.${app_domain_internal}";
+      # By default traffic should route internally
+      'GOVUK_APP_DOMAIN':           value => $app_domain_internal;
+      'GOVUK_APP_DOMAIN_INTERNAL':  value => $app_domain_internal;
+      # We do not host Licensify in AWS, so need to redirect to
+      # it's external domain
+      'PLEK_SERVICE_LICENSIFY_URI': value => "https://licensify.${licensify_app_domain}";
+      # Apps that auth with Signon should do so across the publishing domain
```

Comment:
Would be good to have the reason documented here. I can't think of it off the top of my head